### ISSUE TITLE
markdown

### ISSUE THREAD

**Issue by laszewsk** [cybertraining-dsc/fa20-523-337#2]
please review markdown rules, look at our template they implicitly use proper rules

**Comment by dataML007** [comment]
Hello Professor - Sure. I have spend quite a bit of time on trying to get the repo as close to the template as well. This has been a challenging task for us. Also thanks for understanding and extending the deadlines. I am working on this to fix this issue.

Thanks,
Balaji

**Comment by laszewsk** [comment]
We gave a detailed template. I do not understand why that would cause an issue?

**Comment by dataML007** [comment]
Hello Professor - Yes, I followed the template. However, there was still issues (even now!) for my references. I spend hours trouble shooting it along with TA's as well. I will reach out to them again this week. I am not sure why "References" aren't showing up? I have used the sample markdown and also check other team's references and ensured mine is the same format. However for some reason my references aren't showing up in our report. Here is our team report:

[My Report](https://github.com/cybertraining-dsc/fa20-523-337/blob/main/project/project.md)
